apps/condo/domains/organization/schema/OrganizationEmployee.test.js

```diff
@@ -529,4 +530,22 @@ describe('OrganizationEmployee', () => {
             phone: client2.userAttrs.phone,
         })
     })
+
+    describe('Constraint', () => {
+        describe('unique_user_and_organization constraint', () => {
```

Comment:
Please add a test case for `condition: 'Q(deletedAt__isnull=True)'`

Reply:
Please softdelete user and try to add same use once more, it should work.

Reply:
> Please softdelete user and try to add same use once more, it should work.

if we soft delete the user, then we will not be able to create an OrganizationEmployee